src/app/core/utils/http-error/pgid-change.error-handler.ts

```diff
@@ -0,0 +1,11 @@
+import { SpecialHttpErrorHandler } from 'ish-core/interceptors/icm-error-mapper.interceptor';
```

Comment:
I don't think that the SpecialHttpErrorHandler is the correct interceptor to react on icm call failure due to outdated pgid. It was introduced to map errors to specific error codes. I think that a custom `HttpInterceptor` is the better selection.